packages/pn-commons/src/components/NotificationDetail/NotificationDetailDocuments.tsx

```diff
@@ -1,3 +1,5 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
We don't use "i18n" in pn-commons. Find another solution (maybe pass translated string to component?)

Reply:
yes, I think we should pass the string already translated to the component